tests/vcs/git/GitCommitBearTest.py

```diff
@@ -100,6 +100,67 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_github_pull_request_temporary_merge_commit(self):
+        self.run_git_command('remote', 'add', 'upstream',
+                             'https://github.com/coala/coala-quickstart.git')
+        run_shell_command('git fetch upstream pull/259/merge:pytest36')
+        run_shell_command('git checkout pytest36')
+        self.assertEqual(self.run_uut(), [])
+
+        run_shell_command('git fetch upstream pull/257/merge:patch-1')
+        run_shell_command('git checkout patch-1')
+        self.assertEqual(self.run_uut(),
+                         ["Shortlog of HEAD commit isn't in imperative"
+                          " mood! Bad words are 'Fixed'"])
+
+        self.git_commit('Simple git commit')
+        self.assertEqual(self.run_uut(), [])
+
+    def test_github_PR_merge_commit_offline(self):
+        run_shell_command('touch testfile1.txt')
+        run_shell_command('git add testfile1.txt')
+        run_shell_command('git commit -m "First commit"')
+        commit_hash1, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash1 = commit_hash1.strip('\n')
+
+        run_shell_command('git checkout -b feature1 master')
+        run_shell_command('touch testfile2.txt')
+        run_shell_command('git add testfile2.txt')
+        run_shell_command('git commit -m "Second commit"')
+        commit_hash2, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash2 = commit_hash2.strip('\n')
+
+        run_shell_command('git checkout master')
+        run_shell_command('git merge --no-ff feature1')
+        command = ('git commit --amend -m "Merge %s into %s "'
```

Comment:
> `"Merge %s into %s "`

I'm pretty sure that there should be no a space at the end of the commit shortlog